frontend/public/components/resource-quota.jsx

```diff
@@ -13,6 +13,18 @@ const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
 
+const compareResourceType = (left, right) => {
+  const regex = /\.|\//; //find object types delimited by '.' or '/'. Ex: '.cpu', '.memory', or '/persistentvolumeclaims'
```

Comment:
To avoid repeating the split/reverse/join logic, I'd do something like

```js
// Reverse segments around `.` and `/` delimiters.
const reverseSegments = str => str.split(/\.|\//).reverse.join('');
left = reverseSegments(left);
right = reverseSegments(right);
return left.localCompare(right);
```

I wouldn't worry about case insensitive sorts if we're not sorting humanized values.